libmisc/copydir.c

```diff
@@ -751,7 +751,7 @@ static int copy_file (const char *src, const char *dst,
 		return -1;
 	}
 #endif				/* WITH_SELINUX */
-	ofd = open (dst, O_WRONLY | O_CREAT | O_TRUNC, statp->st_mode & 07777);
+	ofd = open (dst, O_WRONLY | O_CREAT | O_TRUNC | O_NOFOLLOW, statp->st_mode & 07777);
```

Comment:
I was going to suggest using O_EXCL here instead, but the manpage says NFS before v3 does not support it, so maybe this is best.